src/tree-internal.component.ts

```diff
@@ -61,13 +64,19 @@ export class TreeInternalComponent implements OnInit {
   public isSelected: boolean = false;
   public isRightMenuVisible: boolean = false;
   public isLeftMenuVisible: boolean = false;
+  public api: TreeInternalAPI;
 
   public constructor(@Inject(NodeMenuService) private nodeMenuService: NodeMenuService,
                      @Inject(TreeService) public treeService: TreeService,
                      @Inject(ElementRef) public element: ElementRef) {
   }
 
   public ngOnInit(): void {
+    this.api = new TreeInternalAPI(this);
+    if (_.get(this.tree, 'node.id', '')) {
+      this.treeService.APIs[this.tree.node.id] = this.api;
```

Comment:
@lovery We should change, cause it makes code more convenient - same term (controller or treeController) that refers to the known concept (TreeController)